<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChargeDialog</class>
 <widget class="QDialog" name="ChargeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>867</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>充值手机号</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="phoneEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>金额</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="chargeMoneyEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="startQueryButton">
        <property name="text">
         <string>开始查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startBindButton">
        <property name="text">
         <string>开始绑定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="importCouponButton">
        <property name="text">
         <string>导入卡券</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteCouponButton">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="delSelectionButton">
        <property name="text">
         <string>勾选</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="delInvalidButton">
        <property name="text">
         <string>无效</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="delBoundButton">
        <property name="text">
         <string>已绑定</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="couponListWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
